ci-operator/config/quay/quay-tests/quay-quay-tests-master.yaml

```diff
@@ -18,6 +18,11 @@ images:
         yum install -y yum-utils && yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo && \
         yum -y install terraform \
         yum clean all && rm -rf /var/cache/yum/*
+    RUN curl -s "https://download-installer.cdn.mozilla.net/pub/firefox/releases/122.0/linux-x86_64/en-US/firefox-122.0.tar.bz2" -o "firefox-122.0.tar.bz2" && \
```

Comment:
Suggest to use below url, it won't need to update per latest version
`curl -Lo firefox.tar.bz2 'https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US'`

Reply:
Update.